src/style.css

```diff
@@ -12,9 +13,19 @@ body {
 
 .header {
   height: 100px;
+  text-shadow: 0 0 3px #f00, 0 0 5px #00f;
 }
 
 .recentScore {
   padding: 0;
   margin: 0;
 }
```

Comment:
- [ ] There is a margin between the recent score and the table listing score. Kindly check the screenshot and fix the issue.

Your
![image](https://user-images.githubusercontent.com/53019916/131810151-add720a0-ecce-4d09-b7d5-eaf0ea3d16b0.png)


Wireframe
![image](https://user-images.githubusercontent.com/53019916/131810285-3de9fef9-924d-497b-8f0b-72159e004f35.png)

Reply:
thanks. it's fixed now

Reply:
thanks. it's fixed  now

Reply:
This issue is well resolved! ✔️